irohad/ametsuchi/impl/rocksdb_command_executor.cpp

```diff
@@ -70,7 +72,7 @@ CommandResult RocksDbCommandExecutor::execute(
 
           RolePermissionSet creator_permissions;
           if (do_validation) {
-            auto names = splitId(creator_account_id);
+            auto names = staticSplitId<2ull>(creator_account_id);
```

Comment:
Do we ever need more than 2 parts?

Reply:
In next PR's there are places with 3 and 4 entries.